packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/validation.tsx

```diff
@@ -122,12 +122,8 @@ export const getMaxMin = (
             )
           }
 
-          const defaultLimitMaxAmount = convertBaseToStandard(
-            'FIAT',
-            limitMaxAmount
-          )
-          if (Number(defaultMax.FIAT) > Number(defaultLimitMaxAmount)) {
-            defaultMax.FIAT = defaultLimitMaxAmount
+          if (Number(defaultMax.FIAT) > limitMaxAmount) {
+            defaultMax.FIAT = String(limitMaxAmount)
```

Comment:
amount is already converted to base since it came like that from BE we do not need to convert it again